components/Modal/CreateImage.js

```diff
@@ -129,7 +129,10 @@ class CreateImage extends React.Component {
                     className="col-sm-3 control-label"
                   ><FormattedMessage defaultMessage="Blueprint" /></label>
                   <div className="col-sm-9">
-                    <p className="form-control-static">{this.props.blueprint.name}</p>
+                    <p 
+                    className="form-control-static" 
+                    id="cmpsr-form-blueprint"
```

Comment:
This is a case where we want to use `data-id`, since we are adding an identifying attribute for the test.

Here you can just change this to `data-id="cmpsr-form-blueprint"`